stable.txt

```diff
@@ -1 +1 @@
-v1.12.1
+v1.12.1
```

Comment:
I would suggest not changing this file since it will frequently conflict and change.

Reply:
Sure will update the PR.

Reply:
I think the version got updated to v.1.12.2. So will resolve the conflict.